responses/answers/link.md

```diff
@@ -1 +1 @@
-https://t.me/joinchat/AAAAADzxg6e6w9xw0EZcNA
```

Comment:
Tehran Ract group isn't the only group this robot handles. There are several more groups including Tehran Linux User Group. 
As we have [implemented a new way to generate temporary links](https://github.com/amin3mej/telegram-observer-bot/blob/master/app/Bot/BotHelper.php#L139-L170) for each group using the same command, we can delete this command. What's your take on this?